### ISSUE TITLE
1.19.3?

### ISSUE THREAD

**Issue by Aces-and-Jacks** [lucaargolo/accurate-maps#7]
Do you plan on updating?

**Comment by lucaargolo** [comment]
Yes, I plan on updating all my mods after Christmas and new year!

**Comment by slushie0** [comment]
Will this mod ever be updated or should i forget about it?

**Comment by Terraphice** [comment]
+1 to seeing this get updated, I really appreciate mods like this that add just a little bit of consistency and visual appeal.
